b2/console_tool.py

```diff
@@ -707,27 +706,19 @@ def run(self, args):
 
 class ListKeys(Command):
     """
-    b2 list-keys [--keyCount <maxKeyCount>] [--startKey <startApplicationKeyId>]
+    b2 list-keys
 
        Lists the application keys for a given account.
-
-       Optionally sets:
-       - the maximum number of keys returned (the default is 100, the max is 10000).
-       - the first key to return when listing keys
     """
 
-    OPTION_ARGS = ['keyCount', 'startKey']
-
     def run(self, args):
-        response = self.api.list_keys(
-            max_key_count=args.keyCount, start_application_key_id=args.startKey
-        )
+        response = self.api.list_keys()
```

Comment:
This will need to list *all* the keys, not just the ones returned from the first call.  It will need to keep calling until there is no `nextApplicationKeyId` in the response.